torch/_inductor/codecache.py

```diff
@@ -1985,15 +1985,29 @@ def compile(
         cuda: bool,
     ) -> str:
         picked_vec_isa = pick_vec_isa()
-        cpp_command = repr(
-            cpp_compile_command(
-                "i",
-                "o",
+
+        from torch._inductor.cpp_builder import (
+            CppBuilder,
+            CppTorchCudaOptions,
+            get_name_and_dir_from_output_file_path,
+        )
```

Comment:
I feel we need some refactoring to avoid the cyclic dep here...

Reply:
@jgong5 Plan to restructure code to solve it in next PR, I have did the exploration: https://github.com/pytorch/pytorch/pull/128330